src/iuse.cpp

```diff
@@ -6910,6 +6905,65 @@ int iuse::holster_ankle(player *p, item *it, bool b, const tripoint &pos)
     return it->type->charges_to_use();
 }
 
+int iuse::belt_loop (player *p, item *it, bool, const tripoint&)
+{
+    // if belt loop empty offer to attach an item otherwise detach the current one
+    if (it->contents.empty()) {
+        // display menu showing only show BELT_CLIP items
+        item *put = &(p->i_at(g->inv_for_flag("BELT_CLIP", _("Attach what to belt loop?"), false)));
+        if (put == NULL || put->is_null()) {
```

Comment:
You can correct them there, but it's not necessary (and may slow down testing).

Also another quality thing: using references instead of pointers where possible without adding extra code. Not necessary, but makes the intent of the code clearer (pointers are only needed where they can be null or changed in the code).